AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -345,6 +352,16 @@
         ]
       }
     },
+    {
+      "type": "Microsoft.Authorization/roleAssignments",
```

Comment:
Can you update the bicep template too in this PR?

Reply:
Yes, I am updating the bicep in this pr.